res/welcome.html

```diff
@@ -166,6 +166,7 @@ <h4 class="mx_Header_subtitle">_t("Decentralised, encrypted chat &amp; collabora
                 <div class="mx_ButtonLabel">_t("Create Account")</div>
             </a>
         </div>
+        <!-- BEGIN Ansible: Remove these lines when guest access is disabled -->
```

Comment:
a comment above this saying "because we want this" would be nice, ideally with an issue reference

Reply:
Thanks, added more context.